INN/umbrella-sfpublicpress
# Homepage largo recent posts styling

## Changes

This pull request makes the following changes:

- Adds styling for the homepage largo recent posts widgets in the `homepage bottom` widget area (grid borrowed from City Limits and modified to match design)
- Adds a `.widget-title` mixin to match the styling for the reused widget title in the design

Desktop:
![screencapture-sfpp-test-2020-01-10-12_03_35](https://user-images.githubusercontent.com/18353636/72171640-65ddb000-33a1-11ea-83df-333758860b7a.png)

Mobile:
![screencapture-sfpp-test-2020-01-10-12_04_06](https://user-images.githubusercontent.com/18353636/72171650-68d8a080-33a1-11ea-81de-3dbd2c892178.png)


## Why



For #21

## Testing/Questions

Features that this PR affects:

- Homepage largo recent posts widgets
- Styling for widget titles that will use the lines design


Questions that need to be answered before merging:

- [ ] Do the lines in the `.widget-title` mixin look ok? They look kind of wonky to me on mobile, but not sure what can be done to fix that. Also, we ran into this issue with City Limits, but each widget title needs a link so that we can target the `a` element and add a white background to it to make sure the lines don't mash into the text
- [ ] Any better solution for moving the `More` link to be inline with the right-side posts than what is done here? https://github.com/INN/umbrella-sfpublicpress/blob/30b9df4a1e86f783c7b25522584473a9dad6406b/wp-content/themes/sfpublicpress/homepages/assets/less/homepage.less#L278-L293

Steps to test this PR:

1. Add a largo recent posts widget to the `homepage bottom` widget area with these settings:
 - title: news
 - number of posts: 3
 - avoid duplicates: yes
 - thumbnail: large
 - excerpt: 2 sentences
 - show byline
 - hide date
 - show top term
 - "more" link text: More News
 - "more" link url: any
2. View the frontend and verify the posts look ok on desktop and mobile
3. Repeat #2 with an eye specifically on the widget title
